.travis.yml

```diff
@@ -39,7 +39,7 @@ before_script:
 #  - chmod +x travis-acceptance-tests.sh
```

Comment:
We can probably comment our the emulator component (and the env variable), as we're not using it now and it might save resources.